diagnostic.md

```diff
@@ -73,20 +97,43 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --request POST http://localhost3000/items --header "Content-Type: application/json" --data '{ "content": "DDOS the server", "done": false, "list":1}
 ```
 
 Write an AJAX request to create a single donut on the server using JSON.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = let createItem =  function(event) {
+  event.preventDefault();
+  $.ajax({
+    url: 'http://localhost:3000/items',
+    type: 'POST',
+    data: {
+      'content': $(event.target).find('#content').val(),
+      'done': $(event.target).find('#done').val(),
+      'list_id': $(event.target).find('#list-id').val()
+    }
+  })
+  .done(logResponseBody)
+  .fail(logRequestError);
+};
 ```
 
 Now, rewrite your AJAX request to use FormData. Assume you have already targeted
 the correct HTML form before registering your AJAX function with the form.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = new FormData(event.target);
```

Comment:
Not quite. `createDonut` should be a function that we run. We use `FormData` by running it on `event.target` and storing the result (an object populated with data from the form) inside a variable; then, we pass that object into the AJAX query as the `data` property.